client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/patterns-data.ts

```diff
@@ -2,65 +2,78 @@ import { useMemo } from 'react';
 import type { Pattern } from './types';
 
 const useHeaderPatterns = () => {
-	const header = 'header';
+	const header = {
+		slug: 'header',
```

Comment:
The Category type has optional props because we only use the slug here. @okmttdhr